repos/system_upgrade/el7toel8/actors/forcedefaultboottotargetkernelversion/tests/test_forcedefaultboot.py

```diff
@@ -109,12 +143,45 @@ def impl(path):
     return impl
 
 
+class mocked_logger(object):
```

Comment:
I don't think you need to monkeypatch logger when you actually don't use it (there's no assert related to it).

Reply:
Actually you need because of mocked actor.